src/collective/collectionfilter/vocabularies.py

```diff
@@ -100,6 +101,9 @@ def translate_portal_type(value):
     """Translate the type based on its i18n domain the fti provides."""
     types_tool = plone.api.portal.get_tool('portal_types')
     fti = {}
+    if type(value) == Message:
+        # we need a unicode string, not a Message Object to make it work
+        value = unicode(value)
```

Comment:
sry ... not python 3 compatible ... you have to use `Products.CMFPlone.utils.safe_unicode` to ensure py2/3 compatibility

Reply:
good point, i already had some fear on that.